phantomcss/homepage-drupal.js

```diff
@@ -6,7 +6,7 @@ phantomcss.init({
   libraryRoot: './node_modules/phantomcss'
 });
 
-casper.start('http://127.0.0.1:8080');
+casper.start('http://localhost/haskala/www/');
```

Comment:
(btw, this is something we'll need to figure how to determine the URL by the ``env`` - I tried it, but it still doesn't work)